### DISCUSSION TITLE
SRM Notes for 2024-01-11

### DISCUSSION THREAD

**Discussion by bruno-f-cruz** [harp-tech/protocol#93] [SRM Notes]
### 
- 🚀Small Issue (5min)
- 🚅Medium Issue (10min)
- 🐢Large Issue (30min) 

### Revisit:
- https://github.com/harp-tech/protocol/issues/8 🚀

### New proposals:
- https://github.com/harp-tech/protocol/issues/25 🚀
- https://github.com/harp-tech/protocol/issues/18 🚀
- https://github.com/harp-tech/protocol/issues/17 🚀
- https://github.com/harp-tech/protocol/issues/31 🚀
- https://github.com/harp-tech/protocol/issues/29 🚀

- https://github.com/harp-tech/protocol/issues/22 & https://github.com/harp-tech/protocol/issues/13 🚅
- https://github.com/harp-tech/protocol/issues/30 🚅

**Comment by glopesdev** [comment]
@bruno-f-cruz I feel https://github.com/harp-tech/protocol/issues/25 and https://github.com/harp-tech/protocol/issues/29 will be longer conversations that will not be resolved in a short presentation, so maybe we should move them to the end of the agenda, or into the next meeting?
